.env

```diff
@@ -1,2 +1 @@
-REACT_APP_API_SERVER_URL=http://localhost:8001
```

Comment:
this the only change I see for this PR. Why this PR at all? Also if you remove this line, you need to put it inside .env.development